src/main/java/com/palantir/gradle/versions/VersionsLockPlugin.java

```diff
@@ -278,138 +275,137 @@ public final void apply(Project project) {
+                    // dependencies
+                    // instead of the two configurations below (or apiElements, runtimeElements etc)
+                    conf.getAttributes().attribute(GCV_USAGE_ATTRIBUTE, GcvUsage.GCV_SOURCE);
+                });
 
-        NamedDomainObjectProvider<Configuration> consistentVersionsProduction =
-                project.getConfigurations().register(CONSISTENT_VERSIONS_PRODUCTION, conf -> {
-                    conf.setDescription("Outgoing configuration for production dependencies meant to be used by "
-                            + "consistent-versions");
+        NamedDomainObjectProvider<Configuration> consistentVersionsProduction = project.getConfigurations()
+                .register(CONSISTENT_VERSIONS_PRODUCTION, conf -> {
+                    conf.setDescription(
+                            "Outgoing configuration for production dependencies meant to be used by "
+                                    + "consistent-versions");
                     conf.setVisible(false); // needn't be visible from other projects
                     conf.setCanBeConsumed(true);
                     conf.setCanBeResolved(false);
                     conf.getAttributes().attribute(Usage.USAGE_ATTRIBUTE, internalUsage);
                 });
 
-        NamedDomainObjectProvider<Configuration> consistentVersionsTest =
-                project.getConfigurations().register(CONSISTENT_VERSIONS_TEST, conf -> {
-                    conf.setDescription("Outgoing configuration for test dependencies meant to be used by "
-                            + "consistent-versions");
+        NamedDomainObjectProvider<Configuration> consistentVersionsTest = project.getConfigurations()
+                .register(CONSISTENT_VERSIONS_TEST, conf -> {
+                    conf.setDescription(
+                            "Outgoing configuration for test dependencies meant to be used by "
+                                    + "consistent-versions");
                     conf.setVisible(false); // needn't be visible from other projects
                     conf.setCanBeConsumed(true);
                     conf.setCanBeResolved(false);
                     conf.getAttributes().attribute(Usage.USAGE_ATTRIBUTE, internalUsage);
                 });
 
-        unifiedClasspath.getDependencies()
-                .add(
-                        createConfigurationDependencyWithScope(
-                                project,
-                                consistentVersionsProduction.get(),
-                                GcvScope.PRODUCTION));
-        unifiedClasspath.getDependencies()
-                .add(
-                        createConfigurationDependencyWithScope(project, consistentVersionsTest.get(), GcvScope.TEST));
+        unifiedClasspath
+                .getDependencies()
+                .add(createConfigurationDependencyWithScope(project, consistentVersionsProduction.get(), GcvScope
+                        .PRODUCTION));
```

Comment:
On the whole I think these 4 lines are nicer, but breaking in the middle of the Enum.VARIANT is a bit weird

Reply:
This should be easily fixable.